libs/doc-components/catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: pine-react
```

Comment:
Should this read "pine-doc-components?"

Reply:
I must not have hit save before pushing 😆 

Addressing. Thank you!